packages/portal-proto/src/pages/_app.tsx

```diff
@@ -67,6 +67,11 @@ if (process.env.NEXT_PUBLIC_DD_ENABLED) {
     trackViewsManually: true,
     defaultPrivacyLevel: "mask",
     version: `v${PUBLIC_APP_INFO?.version}-${PUBLIC_APP_INFO?.hash}`,
+    allowedTracingUrls: [
+      "https://gdc.cancer.gov",
+      // Matches any subdomain of cancer.gov, such as https://portal.gdc.cancer.gov
+      /^https:\/\/[^]+\.cancer\.gov/,
```

Comment:
@a-collet I am not sure if this regex is correct, as it errored out when I tested in https://regex101.com/.

below seems to work fine. Please verify it once.
```suggestion
      /^https:\/\/[^\/\s]+\.cancer\.gov/,
```

Reply:
Let's confirm this

Reply:
Hi all! The DD docs have `/^https:\/\/[^\/]+\.my-api-domain\.com/` in the example. Prettier flags this and corrected it to what's on the branch. 
I can confirm in my tests that requests do have the expected headers.

Reply:
Okay sounds good. I verified it further using ES regex flavor and it works.